sound/soc/sof/intel/hda-stream.c

```diff
@@ -500,6 +500,7 @@ int hda_dsp_stream_init(struct snd_sof_dev *sdev)
 	struct hdac_ext_stream *stream;
```

Comment:
fix typos in commit message

Reply:
sorry don't get this.

Reply:
add and strore hda stream number

s/strore/store